frontend/src/documents/Documents.svelte

```diff
@@ -43,6 +34,20 @@
       router.reload();
     }
   }
+
+  /**
+   * Rename the selected document with <F2>.
+   * Submit with <Enter> and exit with <Escape>.
+   */
+  function keyup(ev: KeyboardEvent) {
```

Comment:
**Note:** This function was moved to satisfy the linter, as `move()` must be defined above this function now.